MainApp/public/app/controllers/gamePageCtrl.js

```diff
@@ -1,21 +1,24 @@
-angular.module('gamePageControllers', ['roomServices', 'circleServices', 'refreshServices'])
-    .controller('gamePageCtrl', function ($scope, $http, $routeParams, $timeout, $location, $anchorScroll, $rootScope, $cookies, PatientService, Room, Agent, Circle, GameState, UserStats, Refresh) {
+angular.module('gamePageControllers', ['roomServices', 'circleServices', 'refreshServices', 'scrollingServices'])
+    .controller('gamePageCtrl', function ($scope, $http, $routeParams, $timeout, $location, $anchorScroll, $rootScope,
+                            $cookies, PatientService, Room, Agent, Circle, GameState, UserStats, Refresh, Scrolling) {
         let app = this;
         let blinkTimer;
         let blinkTimer2;
         Refresh.checkRefresh();
         app.username=$rootScope.username;
         (function startButton() {
-            alert("The goal is to save as many patients as possible");
+            alert("The goal is to save as many patients as possible. Please switch to landscape mode if using a " +
```

Comment:
Actually no need to check on width.
http://stackoverflow.com/questions/3514784/what-is-the-best-way-to-detect-a-mobile-device-in-jquery
Check the accepted answer.
P.S: this is a minor comment. So can be done after the demo as well, if deemed necessary.